source/common/http/http2/metadata_interface.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <unordered_map>
+
+namespace Envoy {
+namespace Http {
+namespace Http2 {
+
+/**
+ * Please refer to #2394 or go/envoy-metadata for more info about Envoy METADATA.
+ * TODO(soya3129): refer to Envoy document.
+ */
+const uint8_t METADATA_FRAME_TYPE = 0x4d;
```

Comment:
Can you add some basic comments about what these magic values mean without needing to redirect to the doc? I'm a little confused about which part of this is in the IETF spec, vs. implemented by nghttp2, vs. us?

Reply:
Fixed.